components/Result.tsx

```diff
@@ -1,9 +1,33 @@
 import { Album } from "@/app/utils/discogs.schemas";
+import Image from "next/image";
+import Icon from "./Icon";
 
 export default function Result({ album }: { album: Album }) {
   return (
-    <>
-      <h3>Year: {album.year}</h3>
-    </>
+    album.title && (
```

Comment:
I'd use `album.title ? <div>...</div> : null` here

[here's a quick article on why](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)

Reply:
Makes sense, great resource.